src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -151,7 +157,12 @@ private void OnOutputDataReceived(object sender, DataReceivedEventArgs e)
 
         private void OnProcessExited(object sender, EventArgs e)
         {
-            string exceptionMessage = string.Join(",", _processStdErrDataQueue.Where(s => !string.IsNullOrEmpty(s)));
+            if (_disposing)
+            {
+                // No action needed
+                return;
+            }
+            string exceptionMessage = string.Join(",", _processStdErrDataQueue.ToList().Where(s => !string.IsNullOrEmpty(s)));
```

Comment:
This will be iterated once the call is made. Was the call to `ToList` required here?